### PR TITLE
chore: replace more blues.io with blues.com

### PR DESCRIPTION
**Pull request by CarltonHenderson**
Replace blues.io with blues.com and Blues Wireless with Blues

### CODE REVIEW

**Comment by zfields** [comment]
On several of those lines where you are changing `.io` to `.com` also list Blues Wireless, instead of Blues.

Should we just change them all at once?

**Comment by CarltonHenderson** [comment]
@zfields replaces 'Blues Wireless' with 'Blues'. Should I have used Blues Inc. anywhere?

**Review by zfields — APPROVED** [review]

**Review by zfields — CHANGES_REQUESTED** [review]

File: library.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
-name=Blues Wireless Notecard
+name=Blues Notecard
```
Comment (CarltonHenderson) [inline library.properties:1]:
```suggestion
name=Blues Wireless Notecard
```

File: library.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 version=1.5.1
-author=Blues Wireless
-maintainer=Blues Wireless <info@blues.io>
+author=Blues
+maintainer=Blues <info@blues.com>
```
Comment (CarltonHenderson) [inline library.properties:4]:
```suggestion
author=Blues Wireless
maintainer=Blues <info@blues.com>
```

File: library.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name=Blues Wireless Notecard
 version=1.5.1
 author=Blues Wireless
-maintainer=Blues Wireless <info@blues.io>
+maintainer=Blues <info@blues.com>
```
Comment (CarltonHenderson) [inline library.properties:4]: ok to change the maintainer name and email? @zfields

Comment (zfields) [reply library.properties:4]: Yes. You could even do the author while you're at it.

**Review by zfields — APPROVED** [review]

File: library.properties [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name=Blues Wireless Notecard
 version=1.5.1
 author=Blues Wireless
```
Comment (CarltonHenderson) [inline library.properties:3]:
```suggestion
author=Blues
```

**Pull request merged**